[MRG] <fetch_california_housing dataset description not found> (finished)

kkang2097 opened the pull request:
<!--
Thanks for contributing a pull request! Please ensure you have taken a look at
the contribution guidelines: https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md#pull-request-checklist
-->

#### Reference Issues/PRs
<!--
Example: Fixes #1234. See also #3456.
Please use keywords (e.g., Fixes) to create link to the issues or pull requests
you resolved, so that they will automatically be closed when your pull request
is merged. See https://github.com/blog/1506-closing-issues-via-pull-requests
-->


#### What does this implement/fix? Explain your changes.

Resolves #10555. I added a california_housing.rst file to the dataset description folder, and
added a small update to the dataset index.rst file to include my california_housing.rst description.

#### Any other comments?


<!--
Please be aware that we are a loose team of volunteers so patience is
necessary; assistance handling other issues is very welcome. We value
all user contributions, no matter how minor they are. If we are slow to
review, either the pull request needs some benchmarking, tinkering,
convincing, etc. or more likely the reviewers are simply busy. In either
case, we ask for your understanding during the review process.
For more information, see our FAQ on this topic:
http://scikit-learn.org/dev/faq.html#why-is-my-pull-request-not-getting-any-attention.

Thanks for contributing!
-->

kkang2097 committed:
Add files via upload

kkang2097 committed:
Delete california_housing.rst

kkang2097 committed:
Add files via upload

kkang2097 committed:
Update index.rst

kkang2097 committed:
Update index.rst

kkang2097 committed:
Update index.rst

jnothman commented on sklearn/datasets/descr/california_housing.rst:
```diff
@@ -0,0 +1,26 @@
+
+As described on the original website:
+
+    We collected information on the variables using all the block groups in California from the 1990 Census. In this sample a block group on average includes 1425.5 individuals living in a geographically compact area. Naturally, the geographical area included varies inversely with the population density. We computed distances among the centroids of each block group as measured in latitude and lon ...
```
Please wrap lines at 80 chars

kkang2097 replied:
okay, got it.

jnothman reviewed (commented):
Please mention "Resolves #xxxx" or similar in the PR description so that github can auto-close the issue when this is merged.

jnothman commented on sklearn/datasets/descr/california_housing.rst:
```diff
@@ -0,0 +1,26 @@
+>>> cal_housing_data = fetch_california_housing()
+
+Then the user can do what they please with the data. Note that the data
+has already been read into the compiler, and using a file importer after this step is not necessary.
```
"compiler" should be "interpreter".

jnothman replied:
but I find this line a bit awkward anyway. Do other descriptions have similar?

kkang2097 replied:
Will edit to sound more informational!

kkang2097 committed:
Update california_housing.rst

kkang2097 committed:
Update california_housing.rst

Capped lines at 80 Char. as requested

jnothman commented on sklearn/datasets/descr/california_housing.rst:
```diff
@@ -0,0 +1,36 @@
+
+The California Housing dataset
+================================
+
```
rm excess blank line

jnothman commented on sklearn/datasets/descr/california_housing.rst:
```diff
@@ -0,0 +1,36 @@
+.. _This data set from the original source:
+http://www.dcc.fc.up.pt/~ltorgo/Regression/cal_housing.html
+
+
```
rm excess blank line

jnothman commented on sklearn/datasets/descr/california_housing.rst:
```diff
@@ -0,0 +1,36 @@
+
+The data set can be accessed using Python by importing like this:
+
+
```
rm excess blank line

jnothman reviewed (approved):
It needs to be "Resolves #xxxx" or "Fixes #xxxx" or "Closes #xxxx" for GitHub to recognise it.

LGTM

jnothman commented on sklearn/datasets/descr/california_housing.rst:
```diff
@@ -0,0 +1,36 @@
+function is the data fetching / caching function that downloads the data
+archive from AT&T.
+
+.. _This data set from the original source:
```
I'm not actually sure what you intend here. THis formatting doesn't display the words. See https://18174-843222-gh.circle-artifacts.com/0/doc/datasets/index.html#the-california-housing-dataset.

Just say "The dataset is originally from "

jnothman reviewed (changes requested):
Sorry, I take back the LGTM...

Most of the other datasets list the Attributes and basic statistics of the dataset. This should too.

kkang2097 committed:
Update california_housing.rst

Removed excess lines, updated documentation to be similar to that of the other datasets.

jnothman reviewed (approved):
Much better

lesteve commented on sklearn/datasets/descr/california_housing.rst:
```diff
@@ -0,0 +1,47 @@
+This data set contains a set of housing information taken 
+in the 1990 Census (in the US). The :func:`sklearn.datasets.fetch_california_housing`
+function is the data fetching / caching function that downloads the data
+archive from AT&T.
```
What does AT&T have todo with California Housing?

kkang2097 replied:
AT&T has the dataset in storage I believe.

lesteve commented on sklearn/datasets/descr/california_housing.rst:
```diff
@@ -0,0 +1,47 @@
+in the 1990 Census (in the US). The :func:`sklearn.datasets.fetch_california_housing`
+function is the data fetching / caching function that downloads the data
+archive from AT&T.
+.. _This data is originally from :
```
I don't think you want to use `..` here.

kkang2097 committed:
Update plot_ensemble_oob.py

kkang2097 committed:
Update perceptron.py

kkang2097 committed:
Merge pull request #2 from kkang2097/patch-5

Patch 5

kkang2097 committed:
Merge pull request #1 from kkang2097/patch-4

Patch 4

kkang2097 committed:
Update huber.py

kkang2097 committed:
Update gpc.py

kkang2097 committed:
Update coordinate_descent.py

kkang2097 committed:
Update bayesian_mixture.py

kkang2097 committed:
Update gaussian_mixture.py

kkang2097 committed:
Update logistic.py

kkang2097 committed:
Update ensemble.rst

jnothman commented:
@kkang2097, either in this branch or in a new one, please use `git reset --hard 407c031` to recover your last good commit here. If you do it in a different branch, you can then open a new PR and close this one. Thanks.

qinhanmin2014 commented:
resolved in #11319, thanks @kkang2097
